### PR TITLE
[GRO-739] Auction-partnerships2 palette v3 migration 

### PR DESCRIPTION
**Pull request by TMcMeans**
This PR solves: [GRO-739]

Screen Shot 2022-03-15 at 10 31 11 AM

[GRO-739]: https://artsyproduct.atlassian.net/browse/GRO-739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 1]
```diff
@@ -1,9 +1,378 @@
...
-      <Text variant="xl">Auction Partnerships App</Text>
+      <MetaTags
+        title="Auction Partnerships"
+        description=""
+        pathname="/auction-partnerships"
+      />
+
+      <Join
+        separator={
+          <Spacer mt={6} mb={6} border="1px solid" borderColor="black15" />
```
Comment (dzucconi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:26]: You could use a `<Separator my={6} />` here

Comment (TMcMeans) [reply src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:26]: right! `my` works for both the directions on the y axis

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 2]
```diff
@@ -1,9 +1,580 @@
...
     <>
-      <Text variant="xl">Auction Partnerships App</Text>
+      <MetaTags
+        title="Auction Partnerships"
+        description=""
+        pathname="/auction-partnerships"
+      />
+
+      <Join
+        separator={<Spacer my={6} border="1px solid" borderColor="black15" />}
```
Comment (damassi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:32]: 👍

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 3]
```diff
@@ -1,9 +1,684 @@
...
+const PartnerWithArtsyForm: React.FC = () => {
+  useLoadScript({
+    id: "marketo-form",
+    src: "https://app-ab14.marketo.com/js/forms2/js/forms2.min.js",
+    onReady: () => {
+      // @ts-ignore
+      window.MktoForms2.loadForm(
+        "//app-ab14.marketo.com",
+        "609-FDY-207",
+        SHOULD_WARN_PRODUCTION_ENVIRONMENT
```
Comment (damassi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:127]: can we just use `getENV("NODE_ENV") === 'production'`

Comment (dzucconi) [reply src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:127]: Staging will have a NODE_ENV of production though.

I was thinking we should rename this variable though.

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 4]
```diff
@@ -1,9 +1,684 @@
...
+            LastName: Yup.string().required("Required"),
+            Title: Yup.string(),
+            Website: Yup.string()
+              .url("Invalid website url")
+              .required("Required"),
+            Email: Yup.string()
+              .email("Invalid email address")
+              .required("Required"),
+            B2B_TOU_Active_Consent__c: Yup.boolean().oneOf([true], "Required"),
+            Type__c: Yup.string(),
```
Comment (dzucconi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:173]: We reverse engineered the field names from inspecting the embedded Marketo form submission.

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx (hunk 3 above)
Comment (damassi) [reply src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:127]: ohhh i see i see 👍 np.

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 5]
```diff
@@ -1,9 +1,678 @@
...
+            LastName: "",
+            Title: "",
+            Website: "",
+            Email: "",
+            B2B_TOU_Active_Consent__c: false,
+            Type__c: "Gallery",
+          }}
+          onSubmit={values => {
+            // @ts-ignore
+            const form = window.MktoForms2.allForms()[0]
```
Comment (damassi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:156]: 🎉

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx [hunk 6]
```diff
@@ -1,9 +1,678 @@
...
+}
+
+const BiddingTools: React.FC = () => {
+  // const image = resized(
+  //   "http://files.artsy.net/images/auction-devices/temp.png",
+  //   {
+  //     width: 910,
+  //     height: 403,
+  //   }
+  // )
```
Comment (damassi) [inline src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:366]: Do we still need this?

**Review by damassi — APPROVED** [review]
One little bit of commented out code that needs to be removed, but otherwise looks good to me! Nice work on wrangling this weird formik / 3rd party hybrid form. Haven't seen that before 💯

File: src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx (hunk 6 above)
Comment (TMcMeans) [reply src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:366]: yes, waiting for the image from design.

Comment (TMcMeans) [reply src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx:366]: I left a placeholder commented out until I get an image without a background back from them.

**Review by damassi — APPROVED** [review]
Let's merge and see things on staging @TMcMeans! And then we can follow-up with another PR if anything needs to change.

**Pull request merged**